UnityProject/Assets/Scripts/Disposals/DisposalBin.cs

```diff
@@ -22,7 +22,10 @@ public class DisposalBin : DisposalMachine, IServerDespawn, IExaminable, IChecke
 		const int AUTO_FLUSH_DELAY = 2;
 		const float ANIMATION_TIME = 1.3f; // As per sprite sheet JSON file.
 
+// Ignore never-assigned-to warning as it is assigned via inspector
+#pragma warning disable CS0649
 		[SerializeField] AudioSource rechargeSFX;
```

Comment:
is you do [SerializeField] AudioSource rechargeSFX = null; should get rid of warning

Reply:
Is that preferred over warning disables?

Reply:
I think so yes, i havnt seen warning disables any where else.